app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryFragment.java

```diff
@@ -123,6 +123,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     ButterKnife.bind(this, llLayout);
     presenter.attachView(this);
 
+    networkText = (TextView) llLayout.findViewById(R.id.network_text);
```

Comment:
Which TextView was used before to show this message ? I mean only the font was wrong right. So it must have been shown in another TextView. Why can't we correct it in that TextView ?

Reply:
there was no textview predefined in the xml file earlier.
The textview networkText was created in the LabraryFragment.java file itself. Thats why the formatting and placement was not the same

Reply:
okay!, BDW , can you please remove the casting to ```TextView``` and check if it works. IMO it is redundant but not sure.
Please check.

Reply:
sure. just a minute

Reply:
yes, you are right. The typecasting was not needed.
I will just update the PR.
Should I squash the commits to a single one?

Reply:
Yes, you can although it can be done at the time of merging, but anyways do it and change the commit message to ```chore: Made font similar in LibraryFragment and ZimFileSelectFragment```